external-configuration-store/ExternalConfigurationStore/ExternalConfigurationStore.Cloud/ExternalConfigurationManager.cs

```diff
@@ -1,16 +1,18 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Reactive.Subjects;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using ExternalConfigurationStore.Cloud.SettingsStore;
```

Comment:
Do you need the full namespace if using in the same namespace?

Reply:
According to C# syntax
In the namespace, can "using" SettingStore directly. Out the namespace, "must" using full namespace.

Reply:
Ic, I missed that it is in a separate project.